Introduce_Defects.py

```diff
@@ -238,17 +239,18 @@ def grouper(sequence):
         for i in range(len(delete_list_H)):
                 if cif_line[0] == delete_list_H[i]:
                     for k in range(3):
-                        Cxyz[i,k] = float(cif_line[k+2])
-
+                        cxyz[i,k] = float(cif_line[k+2])
 input.close()
 
 # add hydrogens
 
 with open("added_H.txt","w") as added_H_xyz:
     for i in range (len(H1)):
-        added_H_xyz.write(H1[i] + "     " + "H" + "     " + str(Cxyz[i][0]) + "   "
-          + str(Cxyz[i][1]) + "   " + str(Cxyz[i][2]) + "   "
-          + "0.0000" + "   " + "Uiso" + "   " + "1.0" + "\n")
+            added_H_xyz.write(
```

Comment:
I'm thinking line 249 should be indented less to distinguish from the hanging-indented lines below it
https://www.python.org/dev/peps/pep-0008/#indentation

Otherwise, looks good!